src/services/measures/attitude/client/attitude.js

```diff
@@ -39,24 +99,28 @@ Gauges.Widgets.attitude = {
 					}
 				});
 			})
-			.add( this, this.publishReadyEvent )
+			.add( this, function(sequence) {
+				this.publishReadyEvent(sequence);
```

Comment:
Hey Matt, I'm dubious about this last step...
The publish event should always be the last thing the widget does when it's "open for business"... calling update after publishing a ready event seems to me a bit off.. that means that theoretically an observer could be notified and access an inconsistent state for this widget (since the reset of its values is done after the event is published).
Wouldn't it be better to make it as in the following ?:
```javascript
.add(this, function(sequence) { 
   this.update({attitude: { pitch: 0, roll: 0 }});
})
.add(this, this.publishReadyEvent);
```